ddtrace/appsec/_iast/_ast/ast_patching.py

```diff
@@ -26,30 +26,285 @@
+    "django.contrib.sessions.exceptions",
+    "django.contrib.sessions.image_formats",
+    "django.contrib.sessions.middleware",
+    "django.contrib.sessions.templatetags",
+    "django.contrib.sessions.wagtail_hooks",
+    "django.contrib.sites",
+    "django.contrib.staticfiles.admin",
+    "django.contrib.staticfiles.apps",
+    "django.contrib.staticfiles.checks",
+    "django.contrib.staticfiles.finders",
+    "django.contrib.staticfiles.image_formats",
+    "django.contrib.staticfiles.models",
+    "django.contrib.staticfiles.storage",
+    "django.contrib.staticfiles.templatetags",
+    "django.contrib.staticfiles.utils",
+    "django.contrib.staticfiles.wagtail_hooks",
+    "django.core.cache.backends",
+    "django.core.cache.backends.base",
+    "django.core.cache.backends.filebased",
+    "django.core.cache.backends.locmem",
+    "django.core.cache.utils",
+    "django.core.checks.async_checks",
+    "django.core.checks.caches",
+    "django.core.checks.compatibility",
+    "django.core.checks.compatibility.django_4_0",
+    "django.core.checks.database",
+    "django.core.checks.files",
+    "django.core.checks.messages",
+    "django.core.checks.model_checks",
+    "django.core.checks.registry",
+    "django.core.checks.security",
+    "django.core.checks.security.base",
+    "django.core.checks.security.csrf",
+    "django.core.checks.security.sessions",
+    "django.core.checks.templates",
+    "django.core.checks.translation",
+    "django.core.checks.urls",
+    "django.core.exceptions",
+    "django.core.mail",
+    "django.core.management.base",
+    "django.core.management.color",
+    "django.core.management.sql",
+    "django.core.paginator",
+    "django.core.signing",
+    "django.core.validators",
+    "django.dispatch.dispatcher",
+    "django.template.autoreload",
+    "django.template.backends",
+    "django.template.backends.base",
+    "django.template.backends.django",
+    "django.template.base",
+    "django.template.context",
+    "django.template.context_processors",
+    "django.template.defaultfilters",
+    "django.template.defaulttags",
+    "django.template.engine",
+    "django.template.exceptions",
+    "django.template.library",
+    "django.template.loader",
+    "django.template.loader_tags",
+    "django.template.loaders",
+    "django.template.loaders.app_directories",
+    "django.template.loaders.base",
+    "django.template.loaders.cached",
+    "django.template.loaders.filesystem",
+    "django.template.response",
+    "django.template.smartif",
+    "django.template.utils",
+    "django.templatetags",
+    "django.templatetags.cache",
+    "django.templatetags.i18n",
+    "django.templatetags.l10n",
+    "django.templatetags.static",
+    "django.templatetags.tz",
+    "django.test",
+    "django.urls",
+    "django.urls.base",
+    "django.urls.conf",
+    "django.urls.converters",
+    "django.urls.exceptions",
+    "django.urls.resolvers",
+    "django.urls.utils",
```

Comment:
As mentioned before, we don't need to include submodules if the parent is included

```suggestion
    "django.urls",
```